lib/gpu/host_buffer.cc

```diff
@@ -13,7 +13,8 @@ namespace gpu {
 
 IMPLEMENT_WRAPPERTYPEINFO(flutter_gpu, HostBuffer);
 
-HostBuffer::HostBuffer() : host_buffer_(impeller::HostBuffer::Create()) {}
+HostBuffer::HostBuffer()
+    : host_buffer_(impeller::HostBuffer::Create(nullptr)) {}
```

Comment:
To fix this in a sensible way I need to change the interface on the Dart side to be spawned from a a gpuContext. So feel free to land this break and I'll fix it later.

Reply:
I took a shot at fixing that, seems to compile OK!